### PR TITLE
improved rounding and testing of the results

### PR DESCRIPTION
**Pull request by bubriks**
[#30](https://github.com/Wappsto/wappsto-python/issues/30)

### CODE REVIEW

File: wappsto/connection/network_classes/value.py [hunk 1]
```diff
@@ -412,6 +409,41 @@ def __validate_value_data(self, data_value):
...
+
+        Returns:
+            data_value
+
+        """
+        data_value = decimal.Decimal(str(data_value))
+        number_step = abs(decimal.Decimal(str(self.number_step)))
+
+        result = (data_value % number_step)
+        print(result)
```
Comment (AndreasBomholtz) [inline wappsto/connection/network_classes/value.py]: Remove debug.

File: test/wappsto_test.py [hunk 2]
```diff
@@ -185,22 +185,23 @@ def setup_class(self):
...
+                                                     (3.3, 1, "3"),
+                                                     (3.0, 1, "3"),
+                                                     (3.9, 1, "3"),
+                                                     (3, 2, "2"),
+                                                     (3.999, 2, "2"),
+                                                     (4, 2, "4"),
+                                                     (1, 0.5, "1"),
+                                                     (1.01, 0.02, "1"),
+                                                     (2.002, 0.02, "2"),
+                                                     (2.002, 0.0002, "2.002")])
```
Comment (AndreasBomholtz) [inline test/wappsto_test.py]: Add test with negative number.
